website/docs/getting-started/features.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Features
+sidebar_position: 2
+---
+
+# Features provided by odo
+
+odo follows "create and push" workflow for almost everything. It means, as a user, when you "create" something the information (or manifest) is stored in a configuration file, and then upon doing a "push" it gets created  on the Kubernetes cluster.
+
+One can take an existing git repository and create an odo component from it, which can be pushed to a Kubernetes cluster.
+
+odo helps deploy and link multiple components and services with each other. By using odo, application developers can develop, test, debug and deploy microservices based applications on Kubernetes without having a deep understanding of the platform.
```

Comment:
This phrase could be split in two parts.

The first part is an important feature of odo, which is not found in competitive tools. You could highlight this is an important feature, and give some more details to make understand clearly what is the feature: you can talk about injecting configuration defined by services into the component.

The second part of the paragraph is the tl;dr of the features of odo. I could be placed at the very beginning of the page.

Reply:
Agree. Expanded the services part and moved the tl;dr to top of the section.